framework/godel/config/internal/v0/godel.go

```diff
@@ -76,6 +76,8 @@ type PluginsConfig struct {
 type SinglePluginConfig struct {
 	// LocatorWithResolverConfig stores the locator and the resolver for the plugin.
 	LocatorWithResolverConfig `yaml:",inline,omitempty"`
+	// LocatorWithResolverConfig stores the locator and the resolver for the plugin.
```

Comment:
Comment is copy-pasted -- should be updated to describe the `Override` parameter.